.travis.yml

```diff
@@ -1,17 +0,0 @@
-sudo: required
-language: android
-android:
-  components:
```

Comment:
This change is not required in this commit.

Reply:
Okay

Reply:
You corrected it but then again removed the file. Can you merge your branch with the latest changes to avoid this?